releng/test_config.yaml

```diff
@@ -420,11 +418,10 @@ nodeK8sVersions:
     args:
     - --repo=k8s.io/kubernetes=master
     prowImage: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-master
-  # TODO(1.28): Uncomment this when adding jobs for release-1.28 branch.
-  # beta:
-  #   args:
-  #   - --repo=k8s.io/kubernetes=release-1.27
-  #   prowImage: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.27
+  beta:
+    args:
+    - --repo=k8s.io/kubernetes=release-1.27
+    prowImage: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.27
```

Comment:
@xmudrii do I need to bump this ?

Reply:
I think yes

Reply:
@ameukam This is still pending

Reply:
@xmudrii Just to understand, if I bump the branch, I can leave the tag of the image to 1.27 or add a 1.28 variant for kubekins-e2e ?

Reply:
@ameukam I think we should build a new kubekins-e2e variant as well prior to merging this PR.

Reply:
@xmudrii PR for the new variant.